asset_proof.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * AssetProof class
+ */
+class AssetProof {
+  /**
+    * @param {string} id
+    * @param {number} age
+    * @param {Uint8Array} hash
+    * @param {string} nonce
+    * @param {Uint8Array} signature
+    */
+  constructor(id, age, hash, nonce, signature) {
+    this.id = id;
+    this.age = age;
+    this.hash = hash;
+    this.nonce = nonce;
+    this.signature = signature;
+  }
+
+  /**
+   * @param {gRPCAssestProof} proof
```

Comment:
Not sure, but should it be like this ?
```suggestion
   * @param {proto.rpc.AssetProof} proof
```
or 
just `AssetProof` since there is no `gRPCAssestProof` class.